README.md

```diff
@@ -6,10 +6,86 @@ The Ziggurat implementation for XRPLF's `rippled` nodes.
 
 1. Clone this repository.
 2. Build [rippled](https://github.com/XRPLF/rippled) from source.
-3. Create the `~/.ziggurat/ripple/setup` directories, copy the [validators configuration](https://github.com/XRPLF/rippled/blob/develop/cfg/validators-example.txt) there, and name it `validators.txt`:
-    ```
-    curl --create-dirs --output ~/.ziggurat/ripple/setup/validators.txt https://raw.githubusercontent.com/XRPLF/rippled/develop/cfg/validators-example.txt
-    ```
+3. Create the `~/.ziggurat/ripple/setup` directories. Next, create a file there named `validators.txt` with the following content:
+
+```
+#
+# Default validators.txt
+#
+# This file is located in the same folder as your rippled.cfg file
+# and defines which validators your server trusts not to collude.
+#
+# This file is UTF-8 with DOS, UNIX, or Mac style line endings.
+# Blank lines and lines starting with a '#' are ignored.
+#
+#
+#
+# [validators]
+#
+#   List of the validation public keys of nodes to always accept as validators.
+#
```

Comment:
General comment, I think this can be a separate PR:

These instructions are getting really huge. I would move lots of stuff here into a script, like we did in Algorand project. Check how instructions are simple here: https://github.com/runziggurat/algorand#getting-started

Alternative is:
This file could be put into [xrpl](https://github.com/runziggurat/xrpl)/setup/ directory,
and then we can have a `cp setup/validators.txt ~/.ziggurat......` command here

Reply:
I liked the idea of file in `xrpl/setup`.  Did that, and updated readme accordingly.